Week3/homework/step2-3.js

```diff
@@ -4,10 +4,11 @@
 function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
   // Replace this comment and the next line with your code
   console.log(str, num, result);
```

Comment:
This `console.log` and similar ones in the rest of this step were supposed to be deleted.